packages/curvenote-cli/src/submissions/submit.ts

```diff
@@ -26,6 +26,9 @@ import { getChecksForSubmission } from './check.js';
 import { getGitRepoInfo } from './utils.git.js';
 import * as uploads from '../uploads/index.js';
 
+const CDN_KEY_RE = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/;
```

Comment:
basically this is effectvily a `isUUID()` check? thats function doesn't exist or anything, just suggesting that this is maybe more broadly useful than the cdn key